examples/sentence-transformers-training/paraphrases/readme.md

```diff
@@ -0,0 +1,83 @@
+
+# Paraphrases Training
+
+## Requirements
+
+First, you should install -
+```bash
+pip install -U sentence-transformers
+pip install git+https://github.com/huggingface/optimum-habana.git
+```
+
+
+## Usage
+
+To fine training on Paraphrase  you can do following steps -
```

Comment:
To fine-tune. 
Also, can you explicitly add the pre-training instruction for this task too?

Reply:
same as above

Reply:
same as above. Also adding of information related to distributed training.